booster/library/Booster/CLOptions.hs

```diff
@@ -289,6 +290,15 @@ levelToContext =
                 [ [ctxt| booster>(simplification *|function *)>warning |]
                 ]
             )
+        ,
+            ( "Timing"
+            ,
+                [ [ctxt| request*,booster|kore>rewrite*,success|failure|abort|detail |]
```

Comment:
In order to distinguish requests easily, I've added a new `request <id>` context. @geo2a @jberthold please let me know if this looks ok. If so, will update the docs and the other `levelToContext` mappings accordingly.

Reply:
I thought we already have a `[ceil]` context?? do we want `[startup][ceil]`? what would be the advantage/difference?

Reply:
i just noticed we no longer have the ceil context, @jberthold what was the reason for removing it?

Reply:
As mentioned, the `ceil` had a double meaning, indicating a ceil rule (with ID) and indicating the ceil analysis phase (which could be included in said `startup` phase).
I was planning to amend that `ceil` issue in a follow-up but will wait for your change and what we decide.